content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -192,7 +192,7 @@ spec:
 ### Lifecycle and PersistentVolumeClaim
 
 The key design idea is that the
-[parameters for a volume claim](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ephemeralvolumesource-v1alpha1-core)
+[parameters for a volume claim](/docs/reference/generated/kubernetes-api/v1.27/#ephemeralvolumesource-v1-core)
```

Comment:
Should it be `{{< param "version" >}}` like in the original doc so that it can dynamically find the correct K8s version?

Reply:
This is where the `skew` shortcode should be used.

Reply:
Hi, @xing-yang i have made the changes, PTAL